### DISCUSSION TITLE
7800

### DISCUSSION THREAD

**Discussion by pkerga** [darthcloud/BlueRetro#650] [Q&A]
May i ask for your help ?
I'm trying to build a Blueretro adapter for my atari 7800. Everything works fine except button 2. Can't get it to work.
I've used BlueRetro_parallel_2p.bin 1.8.3 firmware
I checked my connections and solders. Works great with original controler wired.
Here are my ideas:
- I only have a 560 ohm resistor. Could this be explained? But button 1 works... Same thing with 660 ohms...
- I only soldered player port 1
- My esp32 kit is a bit tricky... It is nevertheless an Expressif ESP32. I can only connect initially by adding a usb power supply... maybe a fault with my kit, I'm waiting for new ones

Thanks for your help !

https://zupimages.net/up/23/11/tllo.png
https://zupimages.net/up/23/11/63gl.jpg
https://zupimages.net/up/23/11/a3xh.jpg

**Comment by darthcloud** [comment]
You followed those instructions?

https://github.com/darthcloud/BlueRetro/wiki/BlueRetro-Cables-Build-Instructions#7800-adapter-cable

I never tested it myself yet, I might check net weekend if I find some time

**Reply by pkerga** [reply]
Yes of course, that's my bedside Book !
Works great with 2600 games on 7800. 
Blueretro Controler works like a SMS controler.

![PXL_20230319_135640804](https://user-images.githubusercontent.com/30766052/226181903-a8bcd3cb-6bfd-406a-994a-c6a3b515fc9c.jpg)

**Comment by pkerga** [comment]
Hello
Did you have time to test it ?

**Reply by darthcloud** [reply]
no, sorry
